__init__.py

```diff
@@ -0,0 +1,39 @@
+import requests
+import difflib
+from .utils import unescape
+
+def getTeams():
+    teamMap = {}
+    try:
+        resp = requests.get('http://www.espn.com/college-football/teams').text
```

Comment:
Maybe we should add a timeout to the request? Apparently requests will hang forever if the server doesn't respond.